apps/docs/content/components/Section/index.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: Section
+status: Experimental
+source: https://github.com/primer/brand/tree/main/packages/react/src/Section/Section.tsx
+storybook: '/brand/storybook/?path=/story/components-section--default'
+description: Use the section component to create a semantic section in the page and group related content together.
+a11yReviewed: false
+---
+
+import {PropTableValues} from '../../../src/components'
+import {
+  SectionBackgroundColors,
+  SectionPaddingVariants,
+} from '@primer/react-brand'
+
+```js
+import {Section} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+The default section wraps its content in a semantic section element and provides default paddings.
+
+```jsx live
+<Section>
+  <SectionIntro align="center">
```

Comment:
I don't think the align="center" is needed here. It might also be a little confusing because I initially thought the prop was set on Section, not realisting there was a SectionIntro nested inside. Might be worth switching to Heading to avoid this potential confusion, or drop the align prop altogether.